### ISSUE TITLE
Feature request: Faster site generation by compiling only modified source files

### ISSUE THREAD

**Issue by chetan51** [hyde/hyde-old#33]
If the generator kept a list of compiled files, and only processed new and newer-modified files, the generation time after any small modification to the source files would be greatly reduced.

**Comment by jacqueminv** [comment]
I agree with you chetan51.

Besides, did you have a look at the `-k` option on the command line?

It might help you...

**Comment by chetan51** [comment]
Yeah, I found that option soon after posting this feature request. It's very close to what I requested, so actually I have no problem now! Thanks.
